metadata-ws/src/test/resources/features/common.feature

```diff
@@ -0,0 +1,100 @@
+Feature: Common object
+
+  Scenario: perform options
```

Comment:
The point of this test case is to test CORS, not about being able to server `OPTIONS` requests.